Mathlib/RingTheory/Ideal/Operations.lean

```diff
@@ -1701,6 +1701,40 @@ theorem comap_bot_of_injective : Ideal.comap f ⊥ = ⊥ :=
 
 end Injective
 
+/-- If `f : R ≃+* S` is a ring isomorphism and `I : Ideal R`, then `map f (map f.symm) = I`. -/
```

Comment:
```suggestion
/-- If `f : R ≃+* S` is a ring isomorphism and `I : Ideal R`, then `map f (map f.symm I) = I`. -/
```
also in docs below

I also don't like the name, but that should probably be done in a different PR

bors d+

Reply:
Actually, it should say `map f.symm (map f I) = I`, since that is what the lemma states.